apps/static/index.html

```diff
@@ -874,7 +874,7 @@ <h4>Criador da Extensão da Matrícula</h4>
         'https://addons.mozilla.org/pt-BR/firefox/addon/ufabc-next/';
       const TEACHER_FORM =
         'https://docs.google.com/forms/d/e/1FAIpQLSfbwaJCw-t4SlHJ4akwQNCMNAEBREDcdfrqHs7ROhkuUUwDRQ/viewform?usp=sf_link';
-      const FB_LINK = 'https://api.ufabcnext.com/connect/facebook';
+      const FB_LINK = 'https://www.ufabcnext.com/app/autenticar-facebook';
```

Comment:
não tem uma forma de colocar uma url relativa? ai funcionaria no ambiente de dev também!
```suggestion
      const FB_LINK = '/app/autenticar-facebook';
```

Reply:
infelizmente nao, nao rola. Ja criei a issue rescrevendo essa página para podermos fazer esses ajustes. mas da maneira atual acredito que nao tenha como

Reply:
e se usar o window.location.href? 
```suggestion
      const FB_LINK = `${window.location.href}/app/autenticar-facebook`;
```